docs/source/pages/community/guiding-principles.md

```diff
@@ -0,0 +1,63 @@
+# Guiding Principles
+
+In this document we lay out our vision for
+[**NeuroBlueprint**](https://neuroblueprint.neuroinformatics.dev/)
+and **datashuttle** through three guiding principles.
+
+For additional detail, please refer to our blog posts motivating
+[**NeuroBlueprint**](https://neuroinformatics.dev/blog/neuroblueprint.html)
+and
+[**datashuttle**](https://neuroinformatics.dev/blog/datashuttle.html).
+
+The three guiding principles driving the development of
+**NeuroBlueprint**
+and **datashuttle** are:
+
+## Align with existing community initiatives
+
+[BIDS](https://bids.neuroimaging.io/)
+and
+[NWB](https://www.nwb.org/)
+are the gold-standard for standardising systems neuroscience projects.
+Alignment with these specifications should always be the ultimate goal for any project.
+
```

Comment:
I would disagree with both these points. NWB and BIDS are excellent, but they are not the best choice for every project.

Reply:
Good point, how about:

```
BIDS and NWB are the most comprehensive community standards for systems neuroscience projects.
Adhering to these specifications ensures complete standardisation e.g. in metadata 
and file formats. Though extremely valuable, full compliance with these standards
can be time-consuming and technically difficult.
```

Reply:
Looks good.